src/Microsoft.Azure.SignalR.Common/ServiceConnectionBase.cs

```diff
@@ -41,6 +41,7 @@ internal abstract class ServiceConnectionBase : IServiceConnection
         private bool _isStopped;
         private long _lastReceiveTimestamp;
         protected ConnectionContext _connection;
+        protected string _errorMessage;
```

Comment:
Any reason this isn't private?

Reply:
Inheritance class which override `WriteAsync` can use `_errorMessage`